app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -117,6 +122,15 @@ def create
       respond_for_show(:resource => repository)
     end
 
+    api :GET, "/repositories/repository_types", N_("Show the available repository types")
+    param :creatable, :bool, :desc => N_("When set to 'True' repository types that are creatable will be returned")
+    def repository_types
+      params.permit(:creatable)
```

Comment:
I don't believe this line is necessary either.  The only time you need to use strong params is when passing it into .create() .update() or .new()  righT?